File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/duckdb/FixedPointTypes.scala
package dev.typr.foundationssc.docs.duckdb
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object FixedPointTypes:
  //start
  val decimalType: DuckDbType[BigDecimal] = DuckDbTypes.decimal
  val precise: DuckDbType[BigDecimal] = DuckDbTypes.decimal(18, 6)  // DECIMAL(18,6)
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/duckdb/FixedPointTypes.kt
package dev.typr.foundationskt.docs.duckdb

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*
import java.math.BigDecimal

@Suppress("unused")
class FixedPointTypes {
    //start
    val decimalType: DuckDbType<BigDecimal> = DuckDbTypes.decimal
    val precise: DuckDbType<BigDecimal> = DuckDbTypes.decimalOf(18, 6)  // DECIMAL(18,6)
    //stop
}
